### PR TITLE
Added new widgets

### PR DESCRIPTION
**Pull request by nitinmotgi**

### CODE REVIEW

File: cdap-ui/app/directives/c3/charts.js [hunk 1]
```diff
@@ -171,7 +187,15 @@ ngC3.directive('c3AreaStep', function () {
...
-      scope.initC3(elem, 'area-spline', attr);
+      scope.initC3(elem, 'area-spline', attr, {xtickcount: 5} );
+    }
+  }, baseDirective);
+});
+
+ngC3.directive('c3AreaSplineStacked', function () {
+  return angular.extend({
+    link: function (scope, elem, attr) {
+      scope.initC3(elem, 'area-spline', attr, {stack: true});
```
Comment (anwar6953) [inline cdap-ui/app/directives/c3/charts.js]: Why not xtickcount = 5?

File: cdap-ui/app/features/dashboard/templates/userdashboard.html [hunk 2]
```diff
@@ -3,7 +3,7 @@ <h3 class="text-center"> This dashboard is either deleted or not found.</h3>
 </div>
 
 <div class="well well-lg" ng-if="currentBoard.isEmpty">
-  <h3 class="text-center"> No cool widgets in this board! <a ui-sref="dashboard.user.addwdgt"> Add one! </a>  </h3>
+  <h3 class="text-center"> Add widgets to this board! <a ui-sref="dashboard.user.addwdgt"> Add one! </a>  </h3>
```
Comment (anwar6953) [inline cdap-ui/app/features/dashboard/templates/userdashboard.html]: Redundant message:
"Add widgets to this board! Add One!"

Comment (nitinmotgi) [inline cdap-ui/app/features/dashboard/templates/userdashboard.html]: done!

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**